### PR TITLE
py-particle: add version 0.15.1

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-particle/package.py [hunk 1]
```diff
@@ -20,6 +20,8 @@ class PyParticle(PythonPackage):
     tags = ['hep']
 
     version('master', branch='master')
+    version('0.15.1', sha256='6b05cdc4b76c70f785e89258a470504ad87ca119057c65da30a7d4412cca824f')
+    version('0.15.0', sha256='dbf00b97918b52563ae1d6597231323733229bf5d681834ed82988dcea01a48e')
     version('0.14.1', sha256='05b345f8fbfdb12a0aa744c788b6e1b22326b5a6ad95230596e0fc9ebad56621')
     version('0.11.0', sha256='e90dc36c8b7d7431bd14ee5a28486d28b6c0708555845d1d7bdf59a165405f12')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-particle/package.py:25]: Can you update the required version of hepunits? Should be 2+ now.

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-particle/package.py (hunk 1 above)
Comment (vvolkl) [reply var/spack/repos/builtin/packages/py-particle/package.py:25]: Done

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-particle/package.py:25]: Did you forget to push?

Comment (vvolkl) [reply var/spack/repos/builtin/packages/py-particle/package.py:25]: Indeed, sorry about that!

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**